### DISCUSSION TITLE
Update ESPHOME lxc

### DISCUSSION THREAD

**Discussion by bdollerup** [tteck/Proxmox#3987] [Help Needed] [answered]
When following the instruction on the https://tteck.github.io/Proxmox/#esphome-lxc page, I see the below:

```
 ✓ Stopped ESPHome
 | Updating ESPHome
[ERROR] in line 66: exit code 0: while executing command pip3 install -U esphome &> /dev/null
```

The end result is that the service is no longer running and I need to restart the LXC. I'd much appriciate suggestions as to how to fix this.

Thanks

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/3300
